tests/GUIUtils.lua

```diff
@@ -99,18 +99,21 @@ describe("Tests the GUI utilities as far as possible without mudlet", function()
         { "\27[38;2;120;134;94m", "<120,134,94:>"},
         { "\27[48;2;85;250;33m", "<:85,250,33>"},
         { "\27[38;2;120;134;94;48;2;85;250;33m", "<120,134,94:85,250,33>"},
```

Comment:
This sequence is possibly not what you expect, after the initial `\27[38;2;` all of the following `120;134;94;48;2;85` are parameters for that colour then the next `;250;33` are _to my mind_ discarded before the `m`.  This is probably why the `:` was originally specified so that the sequence could be (___I think___):
`\27[38:2:120:134:94;48:2:85:250:33m`